static/app/views/performance/browser/webVitals/utils/queries/storedScoreQueries/calculatePerformanceScoreFromStored.tsx

```diff
@@ -28,6 +28,13 @@ function getTotalScore(data: TableDataRow): number {
   return data[`avg(measurements.score.total)`] as number;
 }
 
+function hasWebVitalScore(data: TableDataRow, webVital: WebVitals): boolean {
+  if (Object.keys(data).includes(`count_scores(measurements.score.${webVital})`)) {
```

Comment:
I'd say it's preferred to using hasOwnProperty over created the Object.keys array and checking if in the array.
```suggestion
  if (data.hasOwnProperty(`count_scores(measurements.score.${webVital})`)) {
```

Reply:
Updated!